app/views/study/create.scala

```diff
@@ -14,7 +14,7 @@ object create:
 
   def apply(
       data: lila.study.StudyForm.importGame.Data,
-      owner: List[Study.IdName],
+      owner: List[(Study.IdName, Int)],
```

Comment:
this could use a nicer type

Reply:
We only use them in one place, so I think it's ok for now.